### ISSUE TITLE
Testing provides different class label every time

### ISSUE THREAD

**Issue by ranju2015** [GhibliField/Multiclass-Text-Classification-using-BERT#5]
When I test with some examples. I just get different class label every times.

The model generates different logits everytime for the same input.

Please suggest.
